plugin/mouse/mouse.py

```diff
@@ -34,6 +34,12 @@
 )
 
 
+@ctx.action_class("main")
```

Comment:
Remind me why we use main again instead of user?

Reply:
`main` is for talent actions without a prefix. `mouse_click`, `key`, etc